openfisca_us/variables/gov/states/mo/tax/income/credits/mo_property_tax_credit/mo_property_tax_credit_housing_payment_test.py

```diff
@@ -0,0 +1,16 @@
+from openfisca_us.model_api import *
+
+class mo_property_tax_credit_housing_payment_test(Variable):
```

Comment:
Also move this out of the MO folder into the general household expenses folder
```suggestion
class pays_property_tax_or_rent(Variable):
```